plugins/hls-fourmolu-plugin/test/Main.hs

```diff
@@ -26,7 +26,7 @@ tests =
         (if cli then "cli" else "lib")
         [ goldenWithFourmolu cli "formats correctly" "Fourmolu" "formatted" $ \doc -> do
             formatDoc doc (FormattingOptions 4 True Nothing Nothing Nothing)
-        , goldenWithFourmolu cli "formats imports correctly" "Fourmolu" "formatted" $ \doc -> do
+        , goldenWithFourmolu cli "formats imports correctly" "Fourmolu2" "formatted" $ \doc -> do
```

Comment:
This seems to have been testing the wrong file for quite a while:
https://github.com/haskell/haskell-language-server/pull/1836/commits/f6535554815a980d936ad2e1822c8bde4155d9a9#diff-5e783ba6072f1ca27c7171a3ece5043ea46f212e57f68272b52ecc36a8152d59L26-L36